wzdx/sample_files/README.md

```diff
@@ -10,6 +10,6 @@ Output Message : WZDx message Generation Output
 
 validation_schema : wzdx schema(from specification)
 
--wzdx_v3.0_feed.json
--wzdx_v3.1_feed.json
--wzdx_v4.0_feed.json
+- wzdx_v3.1_feed.json
```

Comment:
Looks like this v3.1 has been removed, should we drop it from this README as well?

Reply:
removed